sdk/compute/armcompute/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Release History
 
+## v0.3.0 (released)
```

Comment:
Once we integrate our change log verification into the pipeline it will start failing on this format. See https://aka.ms/azsdk/guideline/changelogs for the format.

Reply:
Yeah, this is truly a problem. 
I heard that someone is making the prepare-release script like in DotNet SDK repo.
We will change the format in the changelog and every release PR after this.